scripts/config/pages.js

```diff
@@ -8,6 +8,7 @@ module.exports = [
     ['cashier/payment_agent_listws',            'app/cashier/payment_agent_list',              'default',    'Payment Agent Deposit'],
     ['cashier/payment_methods',                 'app/cashier/payment_methods',                 'default',    'Payment Methods'],
     ['cashier/session_expired',                 'app/cashier/session_expired',                 null,         'Cashier session expired'],
+    ['cashier/dp2p',                            'app/cashier/dp2p',                            'default',    'DP2P'],
```

Comment:
could you place this on line 5 so we follow the alphabetical ordering in `cashier/`?